UI3

garyhtou opened the pull request:
The UI3 ship will comprise the following PRs (all combined into this PR):
- https://github.com/hackclub/hcb/pull/7741
- https://github.com/hackclub/hcb/pull/7773
- Organization home page https://github.com/hackclub/hcb/pull/7105 (was already merged into `main` under the `event_home_page_redesign_2024_09_21` feature flag)
	- https://github.com/hackclub/hcb/pull/8179
- https://github.com/hackclub/hcb/pull/7828

--- 

- Closes #7492
- Closes #7405 

---

**TODO**

BEFORE MERGING, REMOVE UI3 SPECIFIC CODE (THE TODO COMMENT)

manuthecoder commented:
let the testing begin 🧙💣💥

sampoder commented:
A couple of quick things from me:

<img width="239" height="276" alt="Screenshot 2025-07-19 at 11 05 42 PM" src="https://github.com/user-attachments/assets/19cc3fd9-1e63-43ff-be30-e2bc7b2a8fda" />

This feels really empty? Can we put HCB by Hack Club there or something? 

The grant invitation feels like it needs a refresh:

<img width="761" height="158" alt="Screenshot 2025-07-19 at 11 08 59 PM" src="https://github.com/user-attachments/assets/83104ca6-e7ab-4e63-83a7-4403564930d8" />

I'll keep playing around with it over the next few days to find things!

sampoder commented:
Another quick thing!

<img width="283" height="350" alt="Screenshot 2025-07-19 at 11 24 58 PM" src="https://github.com/user-attachments/assets/06802dfb-6747-4e0f-b3ec-d1252a26b067" />

manuthecoder committed:
trncate

manuthecoder committed:
Fix clickable

manuthecoder committed:
Improve homepage transactions

manuthecoder committed:
fix colors

manuthecoder committed:
Fix cards

Luke-Oldenburg committed:
Merge branch 'main' into ui3

garyhtou committed:
Disable configuring new Webauthn on UI3

manuthecoder committed:
Merge branch 'main' into ui3

manuthecoder committed:
fix mobile nav

manuthecoder committed:
fix badges

manuthecoder committed:
Fix cursors

manuthecoder committed:
Fix cursor

manuthecoder committed:
fix reimbursements

manuthecoder committed:
FIX REIMBURSEMENTS!

manuthecoder committed:
`bin/lint`

sampoder committed:
Merge branch 'main' into ui3

sampoder committed:
Merge branch 'main' into ui3

manuthecoder committed:
[Nav] Adjust footer width for better layout consistency

manuthecoder committed:
Fix buttons

manuthecoder committed:
fix padding

manuthecoder committed:
fix badges

manuthecoder committed:
fix checks!

manuthecoder committed:
`bin/lint`

manuthecoder committed:
FIx border

manuthecoder committed:
Fix activities

manuthecoder committed:
fix ledger

manuthecoder committed:
Fix indent

manuthecoder committed:
Fix card grants

manuthecoder committed:
Fix layout issues in card grants, reimbursements, and invoices views

manuthecoder committed:
fix grid

manuthecoder committed:
Fix reimbursements

manuthecoder committed:
Fix reimbursements

manuthecoder committed:
Merge remote-tracking branch 'origin/main' into ui3

manuthecoder committed:
fix margin

manuthecoder committed:
improve button

manuthecoder committed:
Fix buttons

manuthecoder committed:
fix textareas

manuthecoder committed:
who decided to make a backdrop cursor:pointer 💔

manuthecoder committed:
increase gap in action buttons and set max width for edit modal

garyhtou committed:
Merge branch 'main' into ui3

manuthecoder committed:
Improve HCB code CSS

manuthecoder committed:
how does this even fix it

manuthecoder committed:
fix checks

manuthecoder committed:
Fix actions

manuthecoder committed:
Add section and icon to theme actions in command bar

manuthecoder committed:
Fix menus

manuthecoder committed:
Fix container class

manuthecoder committed:
fix padding

polypixeldev committed:
Merge branch 'main' into ui3

Luke-Oldenburg committed:
Merge branch 'main' into ui3

albertchae committed:
UI3 - Try making puma metrics work by probing in puma workers (#12248)

Testing https://github.com/hackclub/hcb/pull/12238 in a non prod
environment (i.e. UI3)

Follow instructions in
https://docs.appsignal.com/ruby/integrations/puma.html#usage-with-preload_app

albertchae committed:
Puma Appsignal - Try removing doppler again (#12249)

Still no puma metrics in appsignal..

albertchae committed:
Revert Puma appsignal config in UI3 (#12250)

Revert 2 puma appsignal related commits

manuthecoder committed:
fix flash

manuthecoder committed:
Fix long flashes

manuthecoder committed:
fix alert

manuthecoder committed:
fix flashes

manuthecoder committed:
improve ledger filters

manuthecoder committed:
improve padding

manuthecoder committed:
improve hcb codes!

manuthecoder committed:
Improve disbursements

manuthecoder committed:
Add footer styling for hcb-code sections in check deposit and unknown views

manuthecoder committed:
fix menus

manuthecoder committed:
Improve transition duration for active menu state

manuthecoder committed:
fix margin

manuthecoder committed:
Merge branch 'main' into ui3

manuthecoder committed:
fix: update filterbar class for consistent styling

manuthecoder committed:
fix: add authorization and redirect for event edit tab

manuthecoder committed:
fix color

manuthecoder committed:
fix hcb codes

manuthecoder committed:
fix: update footer section classes for consistency in disbursement and invoice views

manuthecoder committed:
fix: add missing div wrapper for table in admin tools view

manuthecoder committed:
Merge branch 'main' into ui3

manuthecoder committed:
fix

manuthecoder committed:
fix lists

manuthecoder committed:
fix padding

manuthecoder committed:
fix: update banner-container class for conditional padding

YodaLightsabr committed:
Merge branch 'main' into ui3

manuthecoder committed:
Fix settled after UI

manuthecoder committed:
Improve headings

manuthecoder committed:
fix dark mode

manuthecoder committed:
Merge remote-tracking branch 'origin/main' into ui3

manuthecoder committed:
Fix size

manuthecoder committed:
Fix cards

manuthecoder committed:
fix card grants

manuthecoder committed:
fix cards

manuthecoder committed:
improve stats labels

manuthecoder committed:
improve reimbursements

manuthecoder committed:
fix stats

manuthecoder committed:
improve balance

manuthecoder committed:
improve charts

manuthecoder committed:
fix contractors

manuthecoder committed:
fix team

manuthecoder committed:
refactor check deposit layout for improved styling

garyhtou committed:
Merge branch 'main' into ui3

polypixeldev committed:
Merge branch 'main' into ui3

manuthecoder committed:
fix pagination

manuthecoder committed:
Merge remote-tracking branch 'origin/main' into ui3

polypixeldev committed:
Merge branch 'main' into ui3

manuthecoder committed:
Merge branch 'main' into ui3

Luke-Oldenburg committed:
Merge branch 'main' into ui3

manuthecoder committed:
fix homepage

manuthecoder committed:
fix snow!

manuthecoder committed:
improve dock item

manuthecoder committed:
fix cards

manuthecoder committed:
fix homepage

manuthecoder committed:
fix

manuthecoder committed:
refactor: clean up code formatting and improve button styling

manuthecoder committed:
Merge remote-tracking branch 'origin/main' into ui3

manuthecoder committed:
refactor: update button styling and improve layout in heading and edit views

manuthecoder committed:
Fix invoices

garyhtou committed:
Merge branch 'main' into ui3

garyhtou committed:
Merge branch 'main' into ui3

manuthecoder committed:
improve suborg header

manuthecoder committed:
fix parent events

manuthecoder committed:
fix ledger

manuthecoder committed:
improve popover

manuthecoder committed:
improve admin

manuthecoder committed:
add invoice

manuthecoder committed:
fix margin

manuthecoder committed:
fix invoice admin tool

manuthecoder committed:
fix command bar

manuthecoder committed:
fix receipts

manuthecoder committed:
fix height

manuthecoder committed:
fix command bar

manuthecoder committed:
fix teenager admin tool

manuthecoder committed:
fix command abr

manuthecoder committed:
fix flash

manuthecoder committed:
improve banners

manuthecoder committed:
fix maintenanec banner

manuthecoder committed:
lint

manuthecoder committed:
fix paragraph

manuthecoder committed:
fix margin

manuthecoder committed:
improve card grants

manuthecoder committed:
add unfollow text

manuthecoder committed:
Fix gap

manuthecoder committed:
fix popover

manuthecoder committed:
Merge branch 'main' into ui3

manuthecoder committed:
Improve heading

manuthecoder committed:
improve margins

manuthecoder committed:
fix margin

manuthecoder committed:
fix cards!

manuthecoder committed:
improve cards

manuthecoder committed:
fix actions

manuthecoder committed:
remove background

manuthecoder committed:
Merge remote-tracking branch 'origin/main' into ui3

manuthecoder committed:
upgrade styles

manuthecoder committed:
fix margins

manuthecoder committed:
improve format menu

manuthecoder committed:
fix lint

manuthecoder committed:
improve card grants!

manuthecoder committed:
Merge remote-tracking branch 'origin/main' into ui3

manuthecoder committed:
Fix number stats

manuthecoder committed:
Improve tag chart

manuthecoder committed:
Fix tags

manuthecoder committed:
Fix percentages

manuthecoder committed:
Fix threshold

manuthecoder committed:
fix percent

manuthecoder committed:
change stroke width

manuthecoder committed:
improve label other calculation

manuthecoder committed:
improve appearance of merchant card

manuthecoder committed:
fix tooltips

manuthecoder committed:
Fix margin

manuthecoder committed:
remove decimal

manuthecoder committed:
improve tick formatting

manuthecoder committed:
Fix ticks

manuthecoder committed:
fix react

manuthecoder committed:
fix merchants

manuthecoder committed:
fix numbers

garyhtou committed:
Merge branch 'main' into ui3

manuthecoder committed:
fix ticks

manuthecoder committed:
Fix width

garyhtou committed:
Merge branch 'main' into ui3

Luke-Oldenburg committed:
Merge branch 'main' into ui3

manuthecoder committed:
Fix width

manuthecoder committed:
Merge branch 'main' into ui3

manuthecoder commented on app/views/hcb_codes/_invoice.html.erb:
```diff
@@ -201,6 +185,13 @@
+    <% end %>
   </section>
 </article>
 
```
fixed

manuthecoder committed:
fix height

manuthecoder committed:
fix height

manuthecoder committed:
fix whitespace

manuthecoder committed:
Add spacing

manuthecoder committed:
fix team card

Luke-Oldenburg committed:
Merge branch 'main' into ui3

Luke-Oldenburg committed:
Merge branch 'main' into ui3

manuthecoder committed:
Fix formatting of tickFormatter in Users component

manuthecoder committed:
fix textarea

manuthecoder committed:
fix admin tools

manuthecoder committed:
fix: ensure minimum height for buttons

manuthecoder committed:
fix buttons

manuthecoder committed:
improve heading

manuthecoder committed:
fix balance

manuthecoder committed:
fix icons

manuthecoder committed:
fix: correct placeholder logic in KBarInput component

manuthecoder committed:
Add logo placeholder

manuthecoder committed:
add logo placeholder

manuthecoder committed:
fix transfer button

manuthecoder committed:
Fix display

manuthecoder committed:
Fix tags

manuthecoder committed:
improve admin actions

manuthecoder committed:
fix menus

manuthecoder committed:
fix grants

manuthecoder committed:
fix card styles and improve hover effects

Luke-Oldenburg committed:
Merge branch 'main' into ui3

Luke-Oldenburg committed:
Merge branch 'main' into ui3

Luke-Oldenburg committed:
Merge branch 'main' into ui3

Luke-Oldenburg committed:
Fix missing images

manuthecoder committed:
Merge branch 'main' into ui3

manuthecoder committed:
FIx lint

manuthecoder committed:
fix dx

manuthecoder committed:
Fix ticks

manuthecoder committed:
fix margin

manuthecoder committed:
fix margin

manuthecoder committed:
Fix padding

manuthecoder committed:
fix margin

manuthecoder committed:
fix margin in Users component

manuthecoder committed:
improve filter menu

manuthecoder committed:
Fix details

manuthecoder committed:
Fix text

manuthecoder committed:
Fix text color

manuthecoder committed:
Fix color

manuthecoder committed:
Fix css

manuthecoder committed:
improve admin tool

manuthecoder committed:
Fix class

manuthecoder committed:
Fix admin tools

manuthecoder committed:
fix admin tool

YodaLightsabr committed:
Merge branch 'main' into ui3

YodaLightsabr committed:
Update padding on login components

YodaLightsabr committed:
Make card grant send UI easier to  errt

Luke-Oldenburg committed:
Update app/views/application/_banner_container.html.erb

manuthecoder committed:
fix support button

manuthecoder committed:
Merge remote-tracking branch 'origin/main' into ui3

manuthecoder committed:
Fix labels

manuthecoder committed:
fix auth

manuthecoder committed:
FIx lint

manuthecoder committed:
fix gap

manuthecoder committed:
Remove some basscss classes

manuthecoder committed:
Remove basscss classes

manuthecoder committed:
Remove max width classes

manuthecoder committed:
Remove height fit

manuthecoder committed:
remove list reset

manuthecoder committed:
Fix cancel tooltip

manuthecoder committed:
Merge branch 'main' into ui3

manuthecoder committed:
Fix schema

manuthecoder committed:
fix contracts

manuthecoder committed:
Fix UI3 comments

manuthecoder committed:
Fix origins

manuthecoder committed:
Fix webauthn origins

manuthecoder committed:
remove puts

manuthecoder committed:
Improve homepage buttons

manuthecoder committed:
Fix border

manuthecoder committed:
Fix opacity
